contrib/golang/filters/http/source/cgo.cc

```diff
@@ -153,12 +153,11 @@ CAPIStatus envoyGoFilterHttpGetIntegerValue(void* r, int id, void* value) {
   });
 }
 
-CAPIStatus envoyGoFilterHttpLog(void* r, uint32_t level, void* message) {
-  return envoyGoFilterHandlerWrapper(
-      r, [level, message](std::shared_ptr<Filter>& filter) -> CAPIStatus {
-        auto mesg = referGoString(message);
-        return filter->log(level, mesg);
-      });
+static FilterLogger logger;
```

Comment:
Please do not use any raw statics do to init fiasco. See CONSTRUCT_ON_FIRST_USE. Please fix other cases as well.

Reply:
Done, thanks.